src/physics/base/GeantParticle.hh

```diff
@@ -0,0 +1,68 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantParticleDef.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+
+#include "base/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * /brief Stores particle information for the GeantImporter class
+ *
+ * This class provides the same structure seen in the particle.root file
+ * created by the app/geant-exporter external code and it is used to load
+ * said ROOT data into memory by the GeantImporter class.
+ */
+
+class GeantParticle
+{
+  protected:
+    std::string name_;
+    ssize_type  pdg_;
+    real_type   mass_;
+    real_type   charge_;
+    real_type   spin_;
+    real_type   lifetime_;
+    bool        is_stable_;
+
+  public:
+    GeantParticle();
+    GeantParticle(std::string name,
+                  ssize_type  pdg,
+                  real_type   mass,
+                  real_type   charge,
+                  real_type   spin,
+                  real_type   lifetime,
+                  bool        is_stable);
+
+    ~GeantParticle();
+
+    GeantParticle operator()(std::string name,
+                             ssize_type  pdg,
+                             real_type   mass,
+                             real_type   charge,
+                             real_type   spin,
+                             real_type   lifetime,
+                             bool        is_stable);
+
+    std::string name();
+    ssize_type  pdg();
+    real_type   mass();
+    real_type   charge();
+    real_type   spin();
+    real_type   lifetime();
+    bool        is_stable();
+
+    void print();
+};
```

Comment:
I think this class is unnecessarily complex: this is only used as an intermediate class for loading data from your ROOT file. You should just be able to make it a simple struct, right? Your constructors don't do any validation of the input, and the class can be constructed in an incomplete/meaningless state. Also, the `operator()` doesn't fit well with the class design: `operator()` should be a "function-like" behavior. Since `Particle` isn't a verb, it doesn't make sense for this to be a function.

Finally, instead of having a `print`, just define the standard free function `std::ostream& operator<<(std::ostream&, const GeantParticle&)`.

After these changes the class should be much smaller.